laboratory-servlet-api-integration/src/it/scala/com/wixpress/petri/test/LaboratoryPropertiesIT.scala

```diff
@@ -0,0 +1,23 @@
+package com.wixpress.petri.test
+
+import com.wixpress.petri.test.SampleAppRunner.DEFAULT_PATH_TO_WEBAPP
+import org.specs2.mutable.SpecificationWithJUnit
+
+class LaboratoryPropertiesIT extends SpecificationWithJUnit {
+
+  "laboratory.properties file" should {
+    "be resilient to IT tests (remain the same as before instantiating SampleAppRunner)" in {
```

Comment:
"resilient" is an ad-hoc name and means nothing. Just use the description in parentheses which directly address your fix.